tests/test_modeling_tf_common.py

```diff
@@ -171,6 +170,12 @@ def _prepare_for_class(self, inputs_dict, model_class, return_labels=False) -> d
                 for k, v in inputs_dict.items()
             }
 
+        if model_class.__name__ == "TFBartForSequenceClassification":
```

Comment:
I think we are able (and it's better) to remove this one, and taking care of the eos token ids in TF bart model testing file. (As I didn't see the same block in `test_modeling_common.py`)

Reply:
There shouldn't be any modification in the common tests for a PR adding a new model.

Reply:
I agree this should go elsewhere, but I couldn't see a clear healthy path around `test_save_load_after_resize_token_embeddings`.  It programmatically creates `input_ids` that will be by definition incompatible with `TFBartForSequenceClassification`.

The base `input_ids` are compatible via the other updates in the PR, but a constant is applied to those ids in the test.   Open to options.

Reply:
If it's just to fix one test, you can overwrite it in the TFBart model tester.

Reply:
@uglyboxer Isn't your updated version of `TFBartModelTester.prepare_config_and_inputs_for_common` already making the inputs compatible with `TFBartForSequenceClassification`?

As far as I know, we don't have a block corresponding to `if model_class.__name__ == "TFBartForSequenceClassification":` in `test_modeling_common.py` (for `pytorch` models). So I think this block here (added by you) is not necessary - or, at least, we can have the correct input preparation logic inside `test_modeling_tf_bart.py`.

Could you check again?